prow/cluster/jobs/istio/istio.io/istio.istio.io.master.yaml

```diff
@@ -0,0 +1,16 @@
+presubmits:
+
+  istio/istio.io:
+
+    - name: presubmit-master
+      path_alias: istio.io/istio.io
+      decorate: true
+      always_run: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/website-tools:2019-07-25
+            command:
+              - make
+              - prow
```

Comment:
nit: is this something a dev would want to run locally? just wonder if it would make sense to call it `make test` or something since it isn't doing anything prow specific

Reply:
It has to run from within the container. There are already targets for devs to use in that repo.

I plan on cleaning all of this kind of stuff up in the container-based  build world. Then the names of targets will be normalized.